jenkins/test-history/buckets.json

```diff
@@ -0,0 +1,11 @@
+{
+    "gs://kubernetes-jenkins/logs/": {
```

Comment:
Do we plan on storing anything more in these entries? If not, it could just be `"gs://kube_azure_log/": "azure$"`